play/src/i18n/en-US/mapEditor.ts

```diff
@@ -22,10 +25,19 @@ const mapEditor: BaseTranslation = {
         jitsiProperties: {
             label: "Jitsi Room",
             roomNameLabel: "Room Name",
+            jitsiUrl: "Jitsi URL",
+            jitsiUrlPlaceholder: "meet.jit.si",
             roomNamePlaceholder: "Room Name",
             defaultButtonLabel: "Open Jitsi Room",
             audioMutedLabel: "Muted by default",
             moreOptionsLabel: "More Options",
+            trigger: "Trigger",
+            triggerMessage: "Trigger Message",
+            triggerShowImmediately: "Show Immediately",
+            triggerOnClick: "On Click",
+            triggerOnAction: "On Action",
+            closable: "Closable",
+            noPrefix: "No Prefix",
```

Comment:
We'll probably need to make a pass here.

The label of the properties in the TMJ file should not match the text displayed in the form.
I mean, "No Prefix" will speak to no-one. We would probably need another label with an additional help text here.